### PR TITLE
build the same horizontal, vertical cards and main structures. add img

### PR DESCRIPTION
**Pull request by NguyenKhaa**

### CODE REVIEW

File: src/component/cardAuthorBook.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    </li>
+                </ul>
+            </div>
+
+            <button>Order Today</button>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (HuanDoan) [inline src/component/cardAuthorBook.html:34]: Rename this file to `card-author-book`

File: src/component/cardBlog.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        </div>
+        <div class="blog__footer">
+            <button>Read more</button>
+            <p>Author - 23.05.2022</p>
+        </div>
+
+    </div>
+</body>
+
+</html>
```
Comment (HuanDoan) [inline src/component/cardBlog.html:27]: Rename this file to `card-blog.html`

File: src/component/cardStore.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <div class="store__footer">
+
+            <button>Order Today</button>
+
+        </div>
+
+    </div>
+</body>
+
+</html>
```
Comment (HuanDoan) [inline src/component/cardStore.html:31]: Rename this file to `card-store.html`

File: src/scss/components/_Blog.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            font-weight: 700;
+        }
+
+        p{
+            font-family: Inter;
+            font-size: 17px;
+            font-weight: 400;
+        }
+    }
+}
```
Comment (HuanDoan) [inline src/scss/components/_Blog.scss:52]: Rename this file to `_blog.scss`

File: src/scss/components/_authorBook.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        width: 220px;
+        height: 65px;
+        padding: 20px, 53px, 21px, 53px;
+        border: 1px solid $yellow;
+        background-color: $white;
+        font-size: 19px;
+        font-family: Cardo;
+    }
+}
+
```
Comment (HuanDoan) [inline src/scss/components/_authorBook.scss:69]: Rename this file to `_author-book.scss`

File: src/scss/components/_cardStore.scss [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            background-color: $white;
+            color: $blue;
+            font-size: 19px;
+            font-family: Cardo;
+            font-weight: 700;
+        }
+
+
+    }
+}
```
Comment (HuanDoan) [inline src/scss/components/_cardStore.scss:70]: Rename this file to `_card-store.scss`

File: src/scss/components/_cardStore.scss [hunk 7]
```diff
@@ -0,0 +1,70 @@
+.store {
```
Comment (HuanDoan) [inline src/scss/components/_cardStore.scss:1]: It should be `card-store` instead of `store`

File: src/scss/components/_authorBook.scss [hunk 8]
```diff
@@ -0,0 +1,69 @@
+.authorBook{
```
Comment (HuanDoan) [inline src/scss/components/_authorBook.scss:1]: It should be `author-book` instead of `authorBook`

**Review by HuanDoan — COMMENTED** [review]
I left some comments on your PR, pls fix it

File: src/component/cardAuthorBook.html (hunk 1 above)
Comment (NguyenKhaa) [reply src/component/cardAuthorBook.html:34]: Done

File: src/component/cardBlog.html (hunk 2 above)
Comment (NguyenKhaa) [reply src/component/cardBlog.html:27]: Done

File: src/scss/components/_Blog.scss (hunk 4 above)
Comment (NguyenKhaa) [reply src/scss/components/_Blog.scss:52]: Done

File: src/scss/components/_authorBook.scss (hunk 5 above)
Comment (NguyenKhaa) [reply src/scss/components/_authorBook.scss:69]: Done

File: src/scss/components/_cardStore.scss (hunk 6 above)
Comment (NguyenKhaa) [reply src/scss/components/_cardStore.scss:70]: Done

File: src/scss/components/_cardStore.scss (hunk 7 above)
Comment (NguyenKhaa) [reply src/scss/components/_cardStore.scss:1]: Done

File: src/scss/components/_authorBook.scss (hunk 8 above)
Comment (NguyenKhaa) [reply src/scss/components/_authorBook.scss:1]: Done

File: src/index.html [hunk 9]
```diff
@@ -1,14 +1,31 @@
...
+            <p class="intro">Many variations of passages of Lorem Ipsum willing araise alteration in some form.</p>
+            <p class="details">Printed Books</p>
+        </div>
+        <div class="card-store__footer">
+
+            <button>Order Today</button>
+
+        </div>
+
+    </div>
```
Comment (HuanDoan) [inline src/index.html:28]: I think it doesn't need the empty lines there

File: src/scss/components/_Blog.scss [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+            font-weight: 400;
+        }
+    }
+
+    &__footer {
+        display: flex;
+        justify-content: space-around;
+
+        button {
+            font-family: cardo;
```
Comment (HuanDoan) [inline src/scss/components/_Blog.scss:37]: `Cardo` or `cardo`?

File: src/scss/components/_card-store.scss [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+            height: 65px;
+            border: 1px solid $yellow;
+            background-color: $white;
+            color: $blue;
+            font-size: 19px;
+            font-family: Cardo;
+            font-weight: 700;
+        }
+
+
```
Comment (HuanDoan) [inline src/scss/components/_card-store.scss:68]: Remove the unnecessary empty lines here

File: src/scss/components/_container.scss [hunk 12]
```diff
@@ -0,0 +1,16 @@
+.container__big{
+    width: 100%;
+    padding: 150px 310px;
+    // background-color: yellow;
```
Comment (HuanDoan) [inline src/scss/components/_container.scss:4]: Remove unused property

File: src/scss/components/_container.scss [hunk 13]
```diff
@@ -0,0 +1,16 @@
+.container__big{
+    width: 100%;
+    padding: 150px 310px;
+    // background-color: yellow;
+}
+
+.container__small{
+    width: 100%;
+    padding: 145px 310px;
+    // background-color: yellow;
```
Comment (HuanDoan) [inline src/scss/components/_container.scss:10]: Remove unused property

File: src/index.html (hunk 9 above)
Comment (NguyenKhaa) [reply src/index.html:28]: Done

File: src/scss/components/_container.scss (hunk 12 above)
Comment (NguyenKhaa) [reply src/scss/components/_container.scss]: Done

File: src/scss/components/_card-store.scss (hunk 11 above)
Comment (NguyenKhaa) [reply src/scss/components/_card-store.scss]: Done

File: src/scss/components/_container.scss (hunk 13 above)
Comment (NguyenKhaa) [reply src/scss/components/_container.scss]: Done

**Review by HuanDoan — APPROVED** [review]

**Pull request merged**